src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -468,9 +468,9 @@ public function iAmOnTheOrderPage($action, $number)
      */
     public function iShouldBeOnTheOrderPage($action, $number)
     {
-        $this->iAmOnTheOrderPage($action, $number);
+        $order = $this->findOneBy('order', array('number' => $number));
```

Comment:
I guess there's no point in getting order with given number from database and then using only order number :)